README.md

```diff
@@ -154,18 +156,135 @@ curl -v -XPOST https://indoor-maps-api.wrld3d.com/v1/poi/UUID?token=dev_auth_tok
 
 ---
 
-## License
+## Indoor Assets
+
+### Create a new Indoor Asset set for a given Indoor Map and Floor
+
+```sh
+curl -v -XPOST https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entity-sets?token=<dev_auth_token> -F "name=<set_name>"
+```
+
+### Add a new Indoor Asset to a set
+
+```sh
+curl -v -X POST https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entity-sets/<set_id>?token=<dev_auth_token> -F "name=<asset_name>" -F "lat=<lat>" -F "lon=<lon>" -F "orientation=<orientation_in_degrees>" -F "model=<model_id>"
+```
+The `model` parameter should match a `prop_id` from our [prop manifest](https://cdn-resources.wrld3d.com/props/latest/Assets/manifest.json).
+
+### Update an Indoor Asset
+
+```sh
+curl -v -X PUT https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entity-sets/<set_id>/entities/<entity_id>?token=<dev_auth_token> -F "name=<updated_name>" -F "orientation=<orientation_in_degrees>"
+```
+
+### Remove an Indoor Asset
+
+```sh
+curl -v -X DELETE https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entity-sets/<set_id>/entities/<entity_id>?token=<dev_auth_token>
+```
+
+### Bulk process Indoor Assets for a set
+
+Similar to the POI API you can perform bulk operations that can create, update or delete existing entities in a set.
+
+```sh
+curl -v -X POST https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entity-sets/<set_id>/bulk?token=<dev_auth_token> -d '{
+      "create":[
+        {"name":"new", "lat":11.1, "lon":22.2, "orientation":45.0, "model":"circular_table_01"},
+        {"name":"new2", "lat":33.3, "lon":44.4, "orientation":22.5, "model":"circular_table_01"}
+      ],
+      "update":[
+        {"id":5, "name":"updated_name"}
+      ],
+      "delete":[
+        6,
+        7
+      ]
+    }'
+```
+
+### Delete an Indoor Asset Set
+
+```sh
+curl -v -X DELETE https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entity-sets/<set_id>?token=<dev_auth_token>
+```
+
+### Publish an Indoor Asset Set
+
+```sh
+curl -v -XPOST https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entity-sets/<set_id>/publish?token=<dev_auth_token>
+```
+
+For more information on publishing an Indoor Asset Set, see the [Indoor Assets Tutorial](TUTORIAL-ASSETS.md#publishing-your-assets).
+
+### Download an Indoor Asset Set as a GeoJSON
+
+```sh
+curl -v https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entity-sets/<set_id>/entities.geojson?token=<dev_auth_token> > ./entities.geojson
+```
+
+### Importing Indoor Asset Sets
+
```

Comment:
Think we should also add the API for importing a geojson.

Reply:
Took me a while to **not** find it. There's no API for that, the geojson gets parsed by the JavaScript client and then the bulk update endpoint is used.